app/mailers/late_reservation_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class LateReservationMailer < ApplicationMailer
+  def forgot_to_buy_gift_email(user)
+    @user = user
+    @url  = 'http://example.com/login'
```

Comment:
replace this with rails path helpers for the login page - in our case, the login page url is defined by the method `login_url`